src/components/Profile.astro

```diff
@@ -0,0 +1,56 @@
+---
+import { Image } from 'astro:assets';
+import type { Author } from '../pages/themes/_types/index.ts';
+
+interface Props {
+	author: Author;
+}
+
+const { author } = Astro.props;
+---
+
+<div class="flex lg:flex-col flex-wrap items-center gap-4">
+	<Image
+		inferSize
+		src={author.avatar ?? ''}
+		height={48}
+		width={48}
+		class="rounded-full size-12 lg:w-full h-fit border"
+		alt="Your avatar"
+	/>
+	<h1 class="border-astro-gray-500 lg:hidden heading-3">{author.name}</h1>
+	<div class="flex flex-col gap-4 pt-2 lg:pt-8 w-full">
+		<div class="flex items-center justify-between">
+			<small class="code text-astro-gray-200">Joined</small>
+			<small class="text-astro-gray-200">
+			{
+				new Date(author.createdAt).toLocaleDateString('en-US', {
+					year: 'numeric',
+					month: 'long',
+					day: 'numeric',
+				})
+			}
+			</small>
+		</div>
+		<hr class="border-astro-gray-500" />
+		<div class="flex items-center justify-between">
+			<small class="code text-astro-gray-200">GitHub</small>
+			<small class="text-astro-gray-200">
+				<a class="link" href={`https://github.com/${author.username}`}>@{author.username}</a>
+			</small>
+		</div>
```

Comment:
For the same reasons as discussed re: the URLs, I think we should probably omit this:

```suggestion
```